Cargo.toml

```diff
@@ -145,6 +145,7 @@ pallet-assets = { git = "https://github.com/moondance-labs/polkadot-sdk", branch
 pallet-balances = { git = "https://github.com/moondance-labs/polkadot-sdk", branch = "tanssi-polkadot-v1.11.0", default-features = false }
 pallet-identity = { git = "https://github.com/moondance-labs/polkadot-sdk", branch = "tanssi-polkadot-v1.11.0", default-features = false }
 pallet-message-queue = { git = "https://github.com/moondance-labs/polkadot-sdk", branch = "tanssi-polkadot-v1.11.0", default-features = false }
+pallet-multiblock-migrations = { package = "pallet-migrations", git = "https://github.com/moondance-labs/polkadot-sdk", branch = "tanssi-polkadot-v1.11.0", default-features = false }
```

Comment:
Good job renaming it. So we now have 2 different "pallet-migrations" in Cargo.lock, one from moonkit and one from polkadot-sdk. It's amazing that this doesn't break everything, Rust is such a great language.

Reply:
Yeah.. it's far from ideal but indeed amazing that it works